src/Views/Dogs/DogEdit.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useParams } from 'react-router-dom';
+import DogForm from '../../Components/DogCard/DogForm';
+import { updateDog } from '../../services/DogRoute';
+import { getDogsById } from '../../services/DogRoute';
+
+export default function DogEdit() {
+  const [name, setName] = useState('');
+  const [bio, setBio] = useState('');
+  const [image, setImage] = useState('');
+  const [age, setAge] = useState('');
+  const [breed, setBreed] = useState('');
+  const [loading, setLoading] = useState(true);
+  const history = useHistory();
+
+  const params = useParams();
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const resp = await getDogsById(params.id);
+      setName(resp.name);
+      setBio(resp.bio);
+      setImage(resp.image);
+      setAge(resp.age);
+      setBreed(resp.breed);
+      setLoading(false);
+    };
+    fetchData();
```

Comment:
You're correctly fetching the data here but it's not making it into the form inputs below.